models/Question.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let Question = new Schema({
+    id: {
+        type: number
+    },    
```

Comment:
When creating the schema mongoose will auto create the id field so this shouldn't be needed.